src/script.js

```diff
@@ -222,6 +222,24 @@ export function getLocale() : LocaleType {
         if (COUNTRY_LANGS.hasOwnProperty(country) && COUNTRY_LANGS[country].indexOf(lang) !== -1) {
             // $FlowFixMe
             return { country, lang };
+        } else {
+            // We will infer country from language if there is only one possible country match
+            const possibleCountries = Object.keys(COUNTRY_LANGS).filter((possibleCountry) => {
+                const languages = COUNTRY_LANGS[possibleCountry];
+
+                if (languages.length && languages.some(language => language === lang)) {
+                    return true;
+                }
+
+                return false;
+            });
+
+            const inferredCountry = COUNTRY[possibleCountries[0]];
+            const languageKey = Object.keys(LANG).filter(possibleLanguage => LANG[possibleLanguage] === lang)[0];
```

Comment:
Would it be clearer to just do `if (values(LANG).indexOf(lang) !== -1)` to check if it's a valid language?

Reply:
I didn't know that function was available, but that's very helpful! I was trying to find a way to avoid using `$FlowFixMe` or `$FlowIgnore` and I'm not sure how else I can make flow happy in this case

Reply:
+1 for using the `values()` helper function from belter assuming it plays nice with Flow.

Reply:
That's so belter!

Reply:
I ended up changing this whole else clause so I no longer need to do a bunch of weird things to make Flow happy